src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,149 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static final String INDEX_OUT_OF_BOUND_MESSAGE = "Index out of bound";
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> prev;
+        private Node<T> next;
+
+        Node(T item) {
```

Comment:
[When creating a new Node, pass all the fields to the Node constructor.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#when-creating-a-new-node-pass-all-the-fields-to-the-node-constructor)